common/djangoapps/pipeline_mako/templates/static_content.html

```diff
@@ -5,6 +5,7 @@
 from mako.exceptions import TemplateLookupException
 
 from microsite_configuration import microsite
+from microsite_configuration import page_title_breadcrumbs
```

Comment:
It would be really great if we could hide common/microsite_configuration functionality behind openedx/theming, even if just as pass-through calls at this time.